doc/code-tests.md

```diff
@@ -139,6 +139,66 @@ command `npm run test:e2e -- --update-snapshots`. This will produce new
 expected snapshots in the directory /e2e/tests, which you will need to add as
 part of your commit.
 
+### Updating snapshots for the Github Continuous Integration (CI) testing
+
+Sometimes the new behavior will also affect the snapshots for tests that are
+run automatically when pull requests (PR)s are created or updated, or new code
+is merged into main. (Note there is a separate set of snapshots for such
+tests, because the images for some of the tests end up slightly different in
+the Github CI runtime environment than on our local machines.)
```

Comment:
It's not clear from this whether this applies to all snapshots or just those that involve the parameter panes, or....?  Because I don't recall having this issue with Turtle.ts.  Can you explain a little more?  And maybe say how to tell whether this is why the github tests aren't passing?  (Is it just that things pass on your machine but not on github?)

Reply:
I will expand this portion of the doc, but you did not have this issue with Turtle because GitHub can't run WebGL tests :-( -- that's why PR review must always include running tests on the reviewer's local machine, not relying just on Github automatic test runs, sadly.